### ISSUE TITLE
NVDA is reading the content of chart if user press shift+tab on Chrome

### ISSUE THREAD

**Issue by vivek31be** [nvaccess/nvda#9109]
[NVDA-Chart-issue.zip](https://github.com/nvaccess/nvda/files/2715640/NVDA-Chart-issue.zip)

Steps to reproduce:
1) unzip the attached file. Open html with chrome from extracted folder.
2) strat NVDA, make sure NVDA is in browse mode by using keys insert+space.
3) do tab, it will show focus on entire chart and will read about chart.
4) do tab again
5) no do shift+tab, NVDA will not read any chart content but focus is on entire chart

Expected Result - for an HTML NVDA should read html content with tab and shift+tab.

Actual Result - NVDA is not reading chart content with shift+tab but reads with tab only.

**Comment by Adriani90** [comment]
It reads only part of the chart title when pressing shift+tab. This is also the case in Firefox 72.0.2.
However I am not sure if this might be a web authoring issue.
cc: @MarcoZehe do you have any suggestions here?

**Comment by MarcoZehe** [comment]
There are several weird tabindex attributes here, on unlabeled div elements without roles, some stuff is hidden via aria-hidden, etc. If you turn virtual cursor off and tab through it, you'll hear several unlabeled section elements spoken. If you turn browse mode back on, some of these are not read because NVDA does not read these unlabeled divs in browse mode.

The reason why there is a difference when reading on these tab stops is that going forward, NVDA reads the whole document to the end once. If you tab forward and it lands on one of those unlabeled sections, then shift-tab back, it lands on the SVG or its container, and only reads the actual label for the SVG. And that is only part of the document's contents, not the legend stuff that comes after.

In my opinion, this example lacks a lot of semantics like useful roles for those things tab should stop at, and proper structure for those legends. The way it is read now, I have absolutely no idea what this thing is trying to tell me.
